opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json

```diff
@@ -167,18 +176,168 @@
                   "chk_difficult_breathing",
                   "chk_unable_to_suck",
                   "chk_umbilical_discharge",
-                  "chk_no_movement",
                   "chk_convulsions",
                   "chk_skin_rashes",
                   "chk_pale",
-                  "chk_eye_discharge",
                   "chk_severe_abdominal_pain",
-                  "chk_lethargy"
+                  "chk_lethargy",
+                  "chk_urination"
                 ]
               }
             ]
           }
         }
+      },
+      {
+        "key": "neonatal_visited_health_facility",
+        "type": "native_radio",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+        "openmrs_entity_parent": "",
+        "label": "Je, katika siku 3 zilizopita, umempeleka mtoto kwenye kituo cha afya kwa mojawapo ya hali hizi zilizotambuliwa?",
+        "options": [
+          {
+            "key": "yes",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
We are phasing out the use of openmrs uuids here, please add the same value as the key